blogo-input/posts/votedelegation.md

```diff
@@ -75,19 +77,19 @@ When Alice wants to delegate to Bob a token for predicate $f$, Alice does the fo
 
 Alice generates a new pair of public and secret keys $(pk',sk')$ for the digital signature scheme (the same used to create the public- and secret- keys of the census Merkle Tree). Observe that the generic SNARK-based anonymous e-voting scheme we described before does not use signatures directly, but still the Merkle Tree consists of public-keys of a digital signature scheme.
 
-Alice signs the string $(vk'||f)$ with secret-key $sk$, that is Alice generates the signature
+Alice signs the string $(pk'||f)$ with secret-key $sk$, that is Alice generates the signature
 
-$$\sigma=Sign(sk,(vk'||f))$$
+$$\sigma=Sign(sk,(pk'||f))$$
 
 Finally, Alice sets as token $T_f$ the following:
 
 $$T_f= (sk', \sigma).$$
 
 Alice can pass the token $T_f$ to Bob and Bob can use it to vote for any option satisfying $f(v,id)=1$ where $id$ is the identifier of the election.
 
-The circuit for the SNARK proof is changed as follows. The public statement consists of the vote $v$, the nullifier but also the function $f$. The witness input by Bob consists of the token $T_f$ and a path in the Merle Tree to $vk$.
+The circuit for the SNARK proof is changed as follows. The public statement is as before. The witness input by Bob will also include the token $T_f$ (that inclues the description of $f$).
```

Comment:
Maybe the word 'circuit' could link to the current circuit spec to provide context on the current design, eg:
```suggestion
The [circuit](https://docs.vocdoni.io/architecture/protocol/anonymous-voting/zk-census-proof.html#zksnark-circuit) for the SNARK proof is changed as follows. The public statement is as before. The witness input by Bob will also include the token $T_f$ (that inclues the description of $f$).
```